info/guide/guide-chapter-3.md

```diff
@@ -345,16 +356,76 @@ try {
 
 Line length should be less than 120 symbols. Otherwise, it should be split.
 
-If `complex property` initializing is too long, it should be split:
+If `complex property` initializing is too long, It should be split into priorities:
+1. Logic Binary Expression (`&&` , `||`)
+2. Comparison Binary Expression (`>` , `<` , `==` , `>` , `=` , `<=` , `!=`)
+3. Other types (Arithmetical and Bit operation) (`+` , `-` , `*` , `/` , `%` , `>>` , `<<` , `*=` , `+=` , `-=` , `/=` , `%=` , `++` , `--` , `in` , `!in` etc)
 
 **Invalid example:**
 ```kotlin
 val complexProperty = 1 + 2 + 3 + 4
 ```
 **Valid example:**
 ```kotlin
-val complexProperty = 1 + 2
-+ 3 + 4
+val complexProperty = 1 + 2 +
+    3 + 4
+```
+
+**Invalid example:**
+```kotlin
+val complexProperty = (1 + 2 + 3 > 0) && ( 23 * 4 > 10 * 6)
+```
+**Valid example:**
+```kotlin
+val complexProperty = (1 + 2 + 3 > 0) &&
+    (23 * 4 > 10 * 6)
+```
+
+Long line which should be split in `Elvis Operator` (`?:`):
+
+**Invalid example:**
+```kotlin
+val value = first ?: second
+```
+**Valid example:**
```

Comment:
Please add a blank line between the code fragments.

Reply:
Done [78342c5](https://github.com/saveourtool/diktat/pull/1384/commits/78342c53dd92a42802f781323a6cc10b03605118)